DevelopGitHub/GitHubExtension/GitHubExtension.Security.WebApi/FrontEnd/src/client/app/stats/stats.route.js

```diff
@@ -0,0 +1,32 @@
+﻿(function () {
+    'use strict';
+
+    angular
+        .module('app.stats')
+        .run(appRun);
+
+    appRun.$inject = ['routerHelper'];
+    /* @ngInject */
+    function appRun(routerHelper) {
+        routerHelper.configureStates(getStates());
+    }
+
+    function getStates() {
+        return [
+            {
+                state: 'stats',
+                config: {
+                    url: '/stats',
+                    templateUrl: 'FrontEnd/src/client/app/stats/stats.html',
+                    controller: 'statsController',
+                    contollerAs: 'vmStats',
+                    title: 'Stats',
+                    settings: {
+                        nav: 4,
+                        content: '<i class="fa fa-lock">Stats</i>'
+                    }
```

Comment:
:+1:  note: if you need some requests to be done before the rendering of your route, you cacn use `resolve` section here.